### PR TITLE
only show section if name translations has changed

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#5176

*Description of changes:*
- was showing blank section with line before
- also, don't show "changed" chip on summary page

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — APPROVED** [review]
Looks good.

Do you want to update the app version so you can tell when your fix is deployed? (Optional but I recommend it.)

**Comment by severinbeauvais** [comment]
Also please check the unit tests :)

**Comment by jordiwes** [comment]
Done & done

File: tests/unit/AlterationSummary.spec.ts [hunk 1]
```diff
@@ -54,7 +54,6 @@ describe('Alteration Summary component', () => {
     expect(wrapper.find(AlterationSummary).exists()).toBe(true)
     expect(wrapper.find(EffectiveDateTime).exists()).toBe(true)
     expect(wrapper.find(ConfirmDialog).exists()).toBe(true)
-    expect(wrapper.find(NameTranslation).exists()).toBe(true)
```
Comment (severinbeauvais) [inline tests/unit/AlterationSummary.spec.ts:57]: Actually, I think AlterationSummary _should_ test whether the NameTranslation sub-component is rendered (depending on various flags of course).

Comment (jordiwes) [reply tests/unit/AlterationSummary.spec.ts]: Ok, I put some name translations in :)

**Pull request merged**